Kickstarter-iOS/Views/Controllers/CommentsViewController.swift

```diff
@@ -14,37 +14,50 @@ private enum Layout {
 internal final class CommentsViewController: UITableViewController {
   // MARK: - Properties
 
-  private lazy var commentComposer: CommentComposerView = {
```

Comment:
Was removing this intentional?

Reply:
`CommentComposerView` is still there, on `line 41`

Reply:
Ah ok I missed that - it should stay up with the other properties though I think, seeing that it's a lazy var and not an accessor.